webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -248,15 +259,7 @@ private async Task ValidateDocumentImportFormAsync(DocumentImportForm documentIm
             }
 
             // Make sure the file type is supported.
-            var fileType = this.GetFileType(Path.GetFileName(formFile.FileName));
-            switch (fileType)
-            {
-                case SupportedFileType.Txt:
-                case SupportedFileType.Pdf:
-                    break;
-                default:
-                    throw new ArgumentException($"Unsupported file type: {fileType}");
-            }
+            _ = this.GetFileType(Path.GetFileName(formFile.FileName));
```

Comment:
Nit: Might want to put a comment here to say this is done to verify file type is supported and that, otherwise, an exception is raise